src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -292,23 +292,23 @@ void ThrusterDynamicEffector::computeForceTorque(double integTime, double timeSt
         }
```

Comment:
1st line of commit message is too long again.

Reply:
Sorry I can't tell which commit this is referring to in Github's interface, but I can trim down the few longest ones. They looked good on my git interface, what character limit are we using for commit messages?

Reply:
Fixed these, note that one commit has added context but does not cut off the actual commit title.